src/Button/Button.component.tsx

```diff
@@ -3,113 +3,602 @@ import * as React from 'react';
 // VENDOR
 import classNames from 'classnames';
 import styled, { css, FlattenSimpleInterpolation } from 'styled-components';
-import { darken } from 'polished';
+import { transparentize } from 'polished';
 // THEME
 import { colors, fonts, sizes } from '../theme';
+import { Colors } from '../Typography/Typography.component';
```

Comment:
I should probably have defined this in the theme import and not in typography. Can we move that to the theme and import it in Typography and Button?